app/uk/gov/hmrc/apiplatformmicroservice/combinedapis/models/JsonFormatters.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformmicroservice.combinedapis.models
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.apiplatformmicroservice.apidefinition.models.ApiCategory
+
+trait BasicCombinedApiJsonFormatters{
+  implicit val apiCategoryFormat: Format[ApiCategory] = Json.format[ApiCategory]
```

Comment:
This formatter also exists in app/uk/gov/hmrc/apiplatformmicroservice/xmlapis/models/JsonFormatters.scala

Reply:
Both xmlApi and CombinedApi use apiCategory so both need the formatter.